tokens/transformation/skia/templates/colors.h.template

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "third_party/skia/include/core/SkColor.h"
+
+namespace leo::light {
+  <%= groupedTokens.light.allTokens.map(prop => {
+    return `constexpr SkColor ${prop.name} = SkColorRGB(${prop.value})`;
```

Comment:
Hi! With all due respect, if this line is for c++ header, should we use `SkColorSetRGB()` instead of `SkColorRGB()`?

Roughly, 
```js
    return `constexpr SkColor ${prop.name} = SkColorSetRGB(${prop.value[0]}, ${prop.value[1]}, ${prop.value[2]})`;
```

Reply:
makes sense